settings_dialog.js

```diff
@@ -79,14 +98,73 @@ function updateInstalledVersions(){
     });    
 }
 
+function checkForInstalledVersions(){
```

Comment:
Why is this needed? There is already `listInstalledVersions` in `install_handler.js`